src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -140,6 +142,10 @@ public void addOptionValue(String option, String value) {
               if (option.equals("sourcepath")) {
                 return;
               }
+              // Cross compilation won't work if we provide javac an empty classpath
+              if (option.equals("classpath") && value.equals("''")) {
+                return;
+              }
```

Comment:
I'll change the previous comment to be `processorpath` instead of `classpath`. But the logic of providing an empty `classpath` (if none was defined) was already present in buck.

Reply:
So why do we need a behavior change in `GroovycStep` along with this change? Was this an existing bug? If so, this should probably be done separately.

Reply:
Tests fail if this is not done. I'll try to refactor to not need this 👍